Handle modules defined in bridge.json the same way as attribute modules

Danielku15 opened the pull request:
Fixes #3335  .

Changes proposed in this pull request:

- Handle modules defined in bridge.json the same way as attribute modules

fabriciomurta commented:
Hello! Thanks for the pull request! It looks good, but the first commit was just unnecessary, it just added some assert() lines because you built in Debug mode.

Wouldn't it be a good idea to have a dedicated Issue3335 integration test for this aspect. Maybe two tiny projects implementing each of the alternatives...

Danielku15 commented:
> It looks good, but the first commit was just unnecessary, it just added some assert() lines because you built in Debug mode.
I was previously working on an older commit of the dev-branch where the asserts/contracts were not included (also not in debug). After I synced again they appeared again, maybe as a result of a bugfix. To ensure that my changes did not have an impact on the output I reverted my changes and first made a commit with the outputs updated. 

> Wouldn't it be a good idea to have a dedicated Issue3335 integration test for this aspect. 
I added 2 dedicated integration tests that ensure the output.
